tests/unit/triagers/plugins/test_shipit.py

```diff
@@ -513,3 +513,66 @@ def test_owner_pr_module_utils_and_modules_updated_submitter_maintainer_2(self):
 
         self.assertEqual(iw.submitter, u'mscherer')
         self.assertFalse(facts[u'owner_pr'])
+
+
+class TestReviewFacts(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
Now that we're on pytest runner please use fixtures instead of inheriting from unittest class. This gives more flexibility and grants access to features unavailable otherwise (like parametrization).